### ISSUE TITLE
Dead or Alive Xtreme : Yellow menu buttons

### ISSUE THREAD

**Issue by thamood** [xemu-project/xemu#1604]
### Title

https://xemu.app/titles/54430007/#Dead-or-Alive-Xtreme-Beach-Volleyball

### Bug Description

Menu buttons background are yellow where it should be transparent white
![Screenshot 2024-03-14 at 4 48 52 PM](https://github.com/xemu-project/xemu/assets/5072835/4911c62b-a6d3-420b-ad13-d9cac731907c)

### Expected Behavior

menus should have transparent background
![Screenshot 2024-03-14 at 4 55 28 PM](https://github.com/xemu-project/xemu/assets/5072835/1e91d834-60ab-4d57-8710-f28f5c5864f3)

### xemu Version

Version 0.7.119

### System Information

Mac mini m2
MacOS 14.3.1

**Comment by jyetcv** [comment]
Looks fixed now.

Version 0.7.119:

![Image](https://github.com/user-attachments/assets/61b4ec02-8176-4f25-9293-6b6cfac1eb5d)

Version 0.8.34:

![Image](https://github.com/user-attachments/assets/638b412b-fd8f-400b-8585-5073bf8e1d47)

**Comment by thamood** [comment]
yes it's fixed but performance became very bad, slow frame rate around 40 fps
